### DISCUSSION TITLE
Channels stuck closing after having my node down for some weeks

### DISCUSSION THREAD

**Discussion by mtelizalde** [lightningnetwork/lnd#8206] [Q&A]
Hello, 
After recovering my node, which was offline for more than one month, I was unable to receive back my sats that were stuck on my lnd channels. Could you please help me recovering my channels sats? I dont understand why the total limbo balance is 0 neither why the local and the remote balance of each channel is 0. 
This is the balance before recovering my node:
![20230906 umbrel status ](https://github.com/lightningnetwork/lnd/assets/58173421/97fe8830-6ccc-4260-ad51-adb97d716c7b)
This is my current node balance:
![20231118 current umbrel node balance](https://github.com/lightningnetwork/lnd/assets/58173421/19bf43e1-0c6c-4f63-b9e9-31e6d15a3c8b)
This is the status of my channels:

    "total_limbo_balance": "0",
    "pending_open_channels": [
    ],
    "pending_closing_channels": [
    ],
    "pending_force_closing_channels": [
    ],
    "waiting_close_channels": [
        {
            "channel": {
                "remote_node_pub": "026165850492521f4ac8abd9bd8088123446d126f648ca35e60f88177dc149ceb2",
                "channel_point": "58b3e4eb94d4219fb51492360c29fc1273a2f9c5fe2d0ecab987e4e5a2f09d72:0",
                "capacity": "500000",
                "local_balance": "0",
                "remote_balance": "0",
                "local_chan_reserve_sat": "0",
                "remote_chan_reserve_sat": "0",
                "initiator": "INITIATOR_LOCAL",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0",
                "chan_status_flags": "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private": true
            },
            "limbo_balance": "0",
            "commitments": {
                "local_txid": "",
                "remote_txid": "",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "0",
                "remote_commit_fee_sat": "0",
                "remote_pending_commit_fee_sat": "0"
            },
            "closing_txid": ""
        },
        {
            "channel": {
                "remote_node_pub": "03391210b7fd47678df727ed7c39ea2f3c8dd43d6fb747582b1c13d4c1376f4714",
                "channel_point": "30df1ed030164965508a4aa15806df0723e42d6d3b809a6cf3a029e8c2366ba1:0",
                "capacity": "1100000",
                "local_balance": "0",
                "remote_balance": "0",
                "local_chan_reserve_sat": "0",
                "remote_chan_reserve_sat": "0",
                "initiator": "INITIATOR_LOCAL",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0",
                "chan_status_flags": "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private": true
            },
            "limbo_balance": "0",
            "commitments": {
                "local_txid": "",
                "remote_txid": "",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "0",
                "remote_commit_fee_sat": "0",
                "remote_pending_commit_fee_sat": "0"
            },
            "closing_txid": ""
        },
        {
            "channel": {
                "remote_node_pub": "035e4ff418fc8b5554c5d9eea66396c227bd429a3251c8cbc711002ba215bfc226",
                "channel_point": "5e8aa72950491a5dedb21307b674ba1e4e58e6f0dd716ee9e00ba7449a637917:1",
                "capacity": "1000000",
                "local_balance": "0",
                "remote_balance": "0",
                "local_chan_reserve_sat": "0",
                "remote_chan_reserve_sat": "0",
                "initiator": "INITIATOR_LOCAL",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0",
                "chan_status_flags": "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private": true
            },
            "limbo_balance": "0",
            "commitments": {
                "local_txid": "",
                "remote_txid": "",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "0",
                "remote_commit_fee_sat": "0",
                "remote_pending_commit_fee_sat": "0"
            },
            "closing_txid": ""
        },
        {
            "channel": {
                "remote_node_pub": "039d4851ad8610065045d872ee300739456f59f09eda02f7536823d17f13b47d8b",
                "channel_point": "379927a5d0f0672b52d57b0a23dd96bf3c10f817f5a4ffe3ec52e7226471f425:1",
                "capacity": "167766",
                "local_balance": "0",
                "remote_balance": "0",
                "local_chan_reserve_sat": "0",
                "remote_chan_reserve_sat": "0",
                "initiator": "INITIATOR_REMOTE",
                "commitment_type": "STATIC_REMOTE_KEY",
                "num_forwarding_packages": "0",
                "chan_status_flags": "ChanStatusRestored",
                "private": true
            },
            "limbo_balance": "0",
            "commitments": {
                "local_txid": "",
                "remote_txid": "",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "0",
                "remote_commit_fee_sat": "0",
                "remote_pending_commit_fee_sat": "0"
            },
            "closing_txid": ""
        }
    ]
}

**Comment by GeorgeTsagk** [comment]
Hey @mtelizalde 

>After recovering my node, which was offline for more than one month

Can you be more specific on what "recovering" means here? Did you use a static channel backup file?

**Comment by mtelizalde** [comment]
Hello!  I meant that I needed to restore my node (I forgot my password). It
took me more than one month to start the recovering process. After syncing
back 100% of the blockchain, I introduced my 24 words seed phrase and
uploaded my static channels backup file that was downloaded on September
6th, since I didnt have a recent one.

At your disposition,
Marcos

**Reply by mtelizalde** [reply]
Hello, Yesterday Nov 24th I received 1,072,059 Sats, but still i havent received yet all the funds I had on my LND channels.
![image](https://github.com/lightningnetwork/lnd/assets/58173421/2d5a7d86-0fb3-4be7-bd99-631a294019dd)

**Reply by GeorgeTsagk** [reply]
It could be possible that your node is broadcasting force-closes with a low fee-rate, and the reason it's taking a while to get your funds back is related to the current high fee environment

**Comment by mtelizalde** [comment]
OK, Thanks for the tip! How many time it could take? I have pacience, I
want just to follow up, keep you posted and ask for support in case I still
don't get back my channels funds.
Is there a way I can check the fee rate that my node used for
force-closings my channels?
